From 37808387b92e05162418e4ec6f74b5c0e30e40e1 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Thu, 20 Nov 2014 08:16:40 -0500
Subject: [redhat] infra: properly re-gen kabi files in rh-release

Without this fixup, a test srpm with kabi additions would see its changes
mistakenly propagated to the next rh-release-based build.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/Makefile b/redhat/Makefile
index e442de1..0d7fead 100644
--- a/redhat/Makefile
+++ b/redhat/Makefile
@@ -152,6 +152,7 @@ rh-release: rh-clean-sources
  @$(REDHAT)/scripts/update_changelog.sh $(KVERSION) $(shell echo $$[$(BUILD) + 1]) $(REDHAT) "$(STAMP_VERSION)-$(PREBUILD)$(shell echo $$[$(BUILD) + 1])$(DIST)$(BUILDID)"
  @git commit -s ../Makefile Makefile.common kernel.spec.template -m "[redhat] kernel-$(STAMP_VERSION)-$(PREBUILD)$(shell echo $$[$(BUILD) + 1])$(DIST)$(BUILDID)"
  @make rh-configs
+ @make rh-kabi
  @make sources-rh
 rh-release-tag:
  @git tag -a -m "kernel-$(STAMP_VERSION)-$(PKGRELEASE)" kernel-$(STAMP_VERSION)-$(PKGRELEASE)
-- 
1.7.1